testutils/parsetestresponse.go

```diff
@@ -0,0 +1,22 @@
+package testutils
+
+import "github.com/dicedb/dicedb-go/wire"
+
+func ParseTestResponse(result *wire.Response) any {
+	if result.Err != "" {
+		return result.Err
+	}
+	switch v:= result.Value.(type){
+	case *wire.Response_VStr:
+		return v.VStr
+	case *wire.Response_VInt:
+		return v.VInt
+	case *wire.Response_VBytes:
+		return v.VBytes
+	case *wire.Response_VFloat:
+		return v.VFloat
+	case *wire.Response_VNil:
+		return v.VNil
+	}
+	return nil
+}
```

Comment:
We should not do this. Instead, I would recommend we check V* types instead of converting it to elemental types. Refer: https://github.com/DiceDB/dice/pull/1508